phpstan.neon

```diff
@@ -49,3 +49,5 @@ parameters:
 
 		- '#Property .* \(App\\Models\\.*(\|null)?\) does not accept (Illuminate\\Database\\Eloquent\\Collection<Illuminate\\Database\\Eloquent\\Model>\|)?Illuminate\\Database\\Eloquent\\Model(\|null)?.#'
 		- '#Property .* \(Illuminate\\Database\\Eloquent\\Collection<App\\Models\\.*>\) does not accept Illuminate\\Database\\Eloquent\\Collection<Illuminate\\Database\\Eloquent\\Model>(\|Illuminate\\Database\\Eloquent\\Model)?.#'
+
+		- '#Access to an undefined property Illuminate\\Contracts\\Auth\\Authenticatable::\$.*.#'
```

Comment:
That explains why PHPStan does not complain. It does complain, but you silenced it. IMHO, we should not do that. What is the whole purpose of PHPStan, if we we suppress any error we do not like?

Reply:
Fixed. :smiley: